### DISCUSSION TITLE
Notetaking equivalent to Loop Habit Tracker? (Open-Source)

### DISCUSSION THREAD

**Discussion by drawtheliner** [iSoron/uhabits#812] [General Discussion]
I'm looking for an offline app which allows fast, lightweight and frictionless simple notetaking with date/time stamp.

I basically want an app as lightweight/simple as Loop Habit Tracker, but for notetaking :) 

Most top rated apps have too many features, which means they're not as frictionless as they could be.

Anyone has any recommendations?

(There is a feature request to add such functionality into Loop, however I understand devs might prefer to keep it focused)

**Comment by iSoron** [comment]
App recommendations are welcome, but to avoid commercial/sponsored spam in our forums, **please restrict your recommendations to open-source applications**. Any non-open-source app recommendation will be removed.

**Comment by liviomendonca** [comment]
Hey there.. for a long time I used Joplin [https://joplinapp.org/](url) it's not thaaaat simple but you don't have to use all its functions and it's quite powerful. It's open source as requested.

Oh.. I replaced Joplin with Tiddlywiki [https://tiddlywiki.com/](url).

[Edit by @iSoron: Removed non-open-source suggestions]
